src/mon/OSDMonitor.cc

```diff
@@ -7902,22 +7902,56 @@ int OSDMonitor::get_crush_rule(const string &rule_name,
   return 0;
 }
 
-int OSDMonitor::check_pg_num(int64_t pool, int pg_num, int size, ostream *ss)
```

Comment:
The commit `mon/OSDMonitor: Make the pg_num check more accurate` lacks the `cherry picked from` info. Use `git cherry-pick -x <commit ID>`. Please also add it's being cherry-picked as dependency for other commits and mention it was reverted.